core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2TokenExchange.java

```diff
@@ -91,6 +91,8 @@ public void setTokenExchangeError(String authIdHash, String message)
 
     public void dropToken(UUID authId)
     {
+        // TODO this may not invalidate ongoing loads (https://github.com/trinodb/trino/issues/10512, https://github.com/google/guava/issues/1881).
```

Comment:
s/may/does/

Reply:
Guava can change. I want the reader to be thoughtful, not take my words for granted.